frontend/src/components/StoreSelectorButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { View, TouchableOpacity, StyleSheet} from "react-native";
+import Icon from 'react-native-vector-icons/MaterialIcons';
+import Svg, { Path } from 'react-native-svg';
+import { useDispatch, connect } from 'react-redux';
+import { actionTypes } from '../constants/ActionTypes';
+import { State } from '../constants/Interfaces';
+
+interface Props {
+    singleStoreActive: boolean
+}
+
+function StoreSelector(props: Props) {
+    let singleStoreStyle: Array<any> = [styles.button, styles.leftButton]
+    let multiStoreStyle: Array<any> = [styles.button, styles.rightButton]
+
+    if (props.singleStoreActive) {
+        singleStoreStyle.push(styles.active);
+    }
+    else {
+        multiStoreStyle.push(styles.active);
+    }
+
+    const dispatch = useDispatch();
+
+    const toggleStoreOpt = () => {
+        dispatch({type: actionTypes.changeStoreOpt, payload: {isSingleStore: !props.singleStoreActive}});
+    }
+
+    return (
+        <TouchableOpacity activeOpacity={0.5} style={{flexDirection: "row", paddingRight: 10}} onPress={toggleStoreOpt}>
+            <View style= {singleStoreStyle}>
+                <Icon name="store" size={25} style={{paddingHorizontal: 8, paddingVertical: 3}}/>
+            </View>
+            <View style= {multiStoreStyle}>
+                <View style={styles.svg}>
+                    <Svg width={30} height={27} viewBox="0 0 30 27" fill="none">
+                        <Path
+                            d="M26.667 9.667H13.333v1.666h13.334V9.667zM27.5 18v-1.667l-.833-4.166H13.333l-.833 4.166V18h.833v5h8.334v-5H25v5h1.667v-5h.833zM20 21.333h-5V18h5v3.333z"
```

Comment:
Did you hard-code these paths yourself? Because that looks like it would be a headache if someone told me to write my own SVGs

Reply:
Nope. I just exported the svg from Figma and then changed the size of the svg